packages/block-library/src/html/edit.js

```diff
@@ -73,11 +73,16 @@ export default function HTMLEdit( { attributes, setAttributes, isSelected } ) {
 					</VisuallyHidden>
 				</>
 			) : (
-				<PlainText
-					value={ attributes.content }
-					onChange={ ( content ) => setAttributes( { content } ) }
-					placeholder={ __( 'Write HTML…' ) }
-					aria-label={ __( 'HTML' ) }
+				<EditorView
+					editorId={'block-library-html__editor'}
+					editorInstructionsText={__(
+						`This editor allows you to input your custom HTML.`
```

Comment:
Seems to be no need for a template literal, it can be a quoted string.

```suggestion
						"This editor allows you to input your custom HTML."
```